build.gradle

```diff
@@ -18,13 +18,13 @@ ext {
     compileSdkVersion = 21
     buildToolsVersion = "21.1.1"
 
-    minSdkVersion = 9
+    minSdkVersion = 15
```

Comment:
Is this variable only used for the samples? It looks like ParseUI-Login and ParseUI-Widget still have `minSdkVersion = 9`. If this is the case, I think it's fine.